src/runtime-rs/src/shim/src/config.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::path::{Path, PathBuf};
+use virtcontainers::config::TomlConfig;
+
+const SANDBOX_CONFIG_PATH_KEY: &str = "io.katacontainers.config_path";
+
+const DEFAULT_CONFIG_PATHS: &[&str] = &[
+    "/etc/kata-containers/configuration.toml",
+    "/usr/share/defaults/kata-containers/configuration.toml",
```

Comment:
Strictly, these are defined in `./src/runtime/Makefile` and this might break in certain packaging scenarios. I think we should either get the `Makefile` to generate a tiny rust stub file containing these values as `const` variables, or use a `build.rs` to determine the values. But since these values are being used by the existing runtime, maybe this should be handled by a higher-level directory and consumed by both the go and rust runtimes?

/cc @devimc, @jcvenegas, @fidencio for thoughts.

Reply:
Leave it to later revisions due to need to generate rust stub file.  I think we may try not to modify `src/runtime` when building `src/runtime-rs`. What about using a seperate Makefile at first ?